tests/sys/fs/fusefs/cache.cc

```diff
@@ -142,7 +142,7 @@ TEST_P(Cache, truncate_by_surprise_invalidates_cache)
 	uint64_t attr_valid, entry_valid;
 	int fd;
 	ssize_t bufsize = strlen(CONTENTS);
-	uint8_t buf[bufsize];
+	uint8_t buf[];
```

Comment:
This is completely wrong. From the previous line it seems that the buffer size is dynamic, but it is not really the case, since `CONTENTS` is defined on line 140 as a fixed string. It would be better to either use the compile-time size of `CONTENTS` (it would have to be redeclared as a `const char []` instead of a `const char *`), or use `strdup()` to make a dynamically allocated copy of the string. In the latter case, it would also have to be freed, of course.